Makefile

```diff
@@ -370,7 +373,13 @@ ifeq ($(REGISTRY), eu.gcr.io)
 	# this is used by the cloud-on-k8s-e2e-tests-ocp job
 	@ gcloud auth configure-docker --quiet
 endif
+ifeq ($(REGISTRY), docker.elastic.co)
+	@ docker login -u $(ELASTIC_DOCKER_LOGIN) -p $(ELASTIC_DOCKER_PASSWORD) push.docker.elastic.co
+	@ docker tag $(E2E_IMG) push.$(E2E_IMG)
+	@ docker push push.$(E2E_IMG)
+else
```

Comment:
We have an issue to "Consider using single, public container registry for ci and dev workflows" which will address this for all build jobs.